robotouille/env.py

```diff
@@ -1,288 +1,20 @@
-from backend.predicate import Predicate
 from backend.object import Object
 from backend.state import State
 from backend.movement.movement import Movement
-from environments.env_generator.builder import entity_to_entity_field, create_unique_and_combination_preds, create_combinations
-import copy
 from domain.domain_builder import build_domain
-from utils.robotouille_utils import trim_item_ID
+from backend.gamemodes.classic import Classic
+from .env_utils import build_identity_predicates, build_location_predicates, build_stacking_predicates, build_goal
```

Comment:
thank you for organizing this better. we'll eventually want to reintegrate PDDL and since these are used by `builder.py` it's good to keep them in one place and switch between creating PDDL predicates or our State predicates.

Reply:
noted!